pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/RuntimeASTTransformer.groovy

```diff
@@ -186,6 +186,7 @@ class RuntimeASTTransformer {
             // Filter for only the desired value type - ModelASTValue for env vars, ModelASTInternalFunctionCall for
             // credentials.
             if (v instanceof ModelASTElement && valueType.isInstance(v) && v.sourceLocation != null) {
+                println(v.sourceLocation.class)
```

Comment:
Ah, yes, lurking `println`s. I miss them so often myself. =)

Reply:
I like this one as it dumps the finished product out to retry. but yes I'll get rid of it